qt/widgets/plotting/src/Mpl/ExternalPlotter.cpp

```diff
@@ -103,24 +102,34 @@ boost::optional<Python::Object> workbenchPlot(QStringList const &workspaceNames,
 } // namespace
 
 namespace MantidQt {
-namespace CustomInterfaces {
+namespace Widgets {
+namespace MplCpp {
 
 ExternalPlotter::ExternalPlotter() {}
 
 ExternalPlotter::~ExternalPlotter() {}
 
+/**
+ * Calls plotSpectra with no kwargs
+ */
+void ExternalPlotter::plotSpectra(std::string const &workspaceName, std::string const &workspaceIndices,
+                                  bool errorBars) {
+  return plotSpectra(workspaceName, workspaceIndices, errorBars, boost::none);
+}
+
 /**
  * Produces an external plot of workspace spectra
  *
  * @param workspaceName The name of the workspace to plot
  * @param workspaceIndices The indices within the workspace to plot (e.g.
  * '0-2,5,7-10')
+ * @param errorBars Boolean value to add/remove erorr bars
```

Comment:
Can we add the `kwargs` as an `@param` in this comment, and same for `plotCorrespondingSpectra`